app/components/ui/PopUpMessage.scss

```diff
@@ -0,0 +1,41 @@
+.pop-up-message {
+  display: table;
+  position: fixed;
+  margin: auto;
+  top: 100px;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  color: white;
+  height: 50px;
+  width: 300px;
```

Comment:
Maybe a min-width makes more sense here? I'm just thinking if the message that is passed in is long, it is going to overflow this 300px.

Reply:
Cool, min-width looks a lot better with large messages. Will use that.